osu.Game/Rulesets/Mods/ModExtensions.cs

```diff
@@ -21,8 +21,10 @@ public static Score CreateScoreFromReplayData(this ICreateReplayData mod, IBeatm
                 {
                     User = new APIUser
                     {
-                        Id = APIUser.SYSTEM_USER_ID,
+                        // TODO: Some fields weren't copied from replayData.User (namely IsBot and Id). Should ModCreatedUser inherit from APIUser so we could pass it verbatim to avoid future mistakes ?
```

Comment:
It should most definitely not. `APIUser` is an overgrown monstrosity. It should be kept as separate from everything else as possible.

```suggestion
```

Just fixing the copy is fine.